package.json

```diff
@@ -37,7 +37,7 @@
     "lodash": "~3.7.0",
     "longjohn": "^0.2.4",
     "mkdirp": "~0.5.0",
-    "mongodb": "^1.4.26",
+    "mongodb": "^2.2.25",
```

Comment:
we are using the lib on version 2+ and the mongo service on 3+. Should we update the mongdb lib version to 3+ as well?